tftrt/examples/cpp/image-classification/README.md

```diff
@@ -0,0 +1,142 @@
+```
```

Comment:
Could you remove the license from the readme?

Reply:
And please add the license to other files.

Reply:
I removed copyright in README added copyrights to other files. Pls note some files are built upon Google code, thus double copyright. Pls review. Thanks.